game_eggs/steamcmd_servers/satisfactory/egg-satisfactory.json

```diff
@@ -15,7 +15,7 @@
         "ghcr.io\/parkervcp\/games:source": "ghcr.io\/parkervcp\/games:source"
     },
     "file_denylist": [],
-    "startup": ".\/Engine\/Binaries\/Linux\/*-Linux-Shipping FactoryGame ?listen -Port={{SERVER_PORT}} -ServerQueryPort={{QUERY_PORT}} -BeaconPort={{BEACON_PORT}} -multihome=0.0.0.0 $(if {{DISABLE_SEASONAL}}; then echo \"-DisableSeasonalEvents\"; fi)",
+    "startup": ".\/Engine\/Binaries\/Linux\/*-Linux-Shipping FactoryGame ?listen -Port={{TELE_PORT}} -ServerQueryPort={{SERVER_PORT}} -BeaconPort={{BEACON_PORT}} -multihome=0.0.0.0 $(if {{DISABLE_SEASONAL}}; then echo \"-DisableSeasonalEvents\"; fi)",
```

Comment:
This change will break any already running server if they update to this egg...

Reply:
I'm not sure if there is any way around this. It would be better to have the correct egg in the repo, though, rather than something that has been wrong from the start.

Reply:
> I'm not sure if there is any way around this. It would be better to have the correct egg in the repo, though, rather than something that has been wrong from the start.

Please refer to these two links:
- [Egg README explaining ports](https://github.com/parkervcp/eggs/tree/master/game_eggs/steamcmd_servers/satisfactory#server-ports) (egg users are expected to read the README before using)
- [Original debate about ports](https://github.com/parkervcp/eggs/pull/1375#issuecomment-954991569) (this, and the two comments below it)

Reply:
To be honest I don't think this PR is needed, it is clearly mentioned in the readme and in the variable description that you have to join with the query port

Reply:
Telling someone to read the docs because you've intentionally made something more difficult than it needs to be isn't going to make things better, though. It's just a bad experience for everyone who is involved.

Reply:
> Telling someone to read the docs because you've intentionally made something more difficult than it needs to be isn't going to make things better, though. It's just a bad experience for everyone who is involved.

that they not read the REAME that is so for most users, but not reading the variable description that is there own fault. This PR can not work and will brake all runing eggs as with an egg update the startup of already made servers will not upadate but the variables will.

Reply:
yes people do that, they just see there is an egg update, they update there egg and then see that there already made server does not work anymore and then they come to us in the discord.

Reply:
If you are going to do anything just swap the query and server ports and don't rename it.

Reply:
When you update an egg it overrides the egg on the old servers. Any server that already exists shouldn't update the start commands. That is a feature that hasn't been implemented yet.

Reply:
> If you are going to do anything just swap the query and server ports and don't rename it.

I'll leave that for someone else to do as it's not correct.

Reply:
updating the variable name is what breaks the currently running servers. You should just leave it named the way it is.

Reply:
I understand, but its just supporting bad practices and will cause confusion with people. I appreciate your time.